### DISCUSSION TITLE
Init mode

### DISCUSSION THREAD

**Discussion by aaHorse** [apache/skywalking#7780] [Q&A]
I have the same problem，here: https://skywalking.apache.org/docs/main/v8.4.0/en/setup/backend/backend-init-mode/
but,use my own oap-images,how can I solve this problem ?

**Comment by wu-sheng** [comment]
What is your problem? You didn't described it. How could we help?

**Comment by aaHorse** [comment]
The official images solves the problem about SkyWalking backend init mode, but we are using our own oap-image.When starting multiple backend instances in k8s, it will not work,how to solve this problem? In addition,we are using Elasticsearch as storage.

**Reply by wu-sheng** [reply]
Still don't know what is the problem. It is your private codes, not community's responsibility.

**Comment by aaHorse** [comment]
Oh, I should learn English well first. Thank you.
